cmd/spiffe-helper/main.go

```diff
@@ -8,7 +8,7 @@ import (
 	"os/signal"
 
 	"github.com/sirupsen/logrus"
-	hclconfig "github.com/spiffe/spiffe-helper/cmd/spiffe-helper/config"
+	config "github.com/spiffe/spiffe-helper/cmd/spiffe-helper/config"
```

Comment:
this name is not required since it is the same than the package

Reply:
Fixed